complaints/models.py

```diff
@@ -65,6 +99,17 @@ def __str__(self):
         return f"{self.user} - {self.category} - {self.id}"
 
     def save(self, *args, **kwargs):
+        if self.bhavan in (self.MEERA, self.MALVIYA):
+            if (
+                self.avail_start_time < self.GIRLS_START_TIME
+                or self.avail_end_time > self.GIRLS_END_TIME
+            ):
+                raise ValidationError(
+                    "Girls Hostel is only open from"
+                    f"{self.GIRLS_START_TIME.strftime('%H:%M')} to "
+                    f"{self.GIRLS_END_TIME.strftime('%H:%M')}."
+                )
+
```

Comment:
Catch these validation errors in the register complaint view also so that these particular errors can be displayed to the user

Reply:
Should I do like we did in unblock request? Catch the error when saving then add it to the response messages

Reply:
Yes